hack/generate/samples/internal/go/generate_v3.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gosamples
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	kbtestutils "sigs.k8s.io/kubebuilder/v2/test/e2e/utils"
+
+	"github.com/operator-framework/operator-sdk/hack/generate/samples/internal/pkg"
+	"github.com/operator-framework/operator-sdk/internal/testutils"
+)
+
+// Run the steps to create the Memcached with Webhooks Go Sample
+func (mh *MemcachedGoWithWebhooks) RunV3() {
+	log.Infof("creating the project")
+	err := mh.ctx.Init(
+		"--plugins", "go/v3-alpha",
+		"--repo", "github.com/example/memcached-operator",
+		"--domain",
+		mh.ctx.Domain)
+	pkg.CheckError("creating the project", err)
+
+	err = mh.ctx.CreateAPI(
+		"--group", mh.ctx.Group,
+		"--version", mh.ctx.Version,
+		"--kind", mh.ctx.Kind,
+		"--controller", "true",
+		"--resource", "true")
+	pkg.CheckError("scaffolding apis", err)
+
+	log.Infof("implementing the API")
+	mh.implementingAPI()
+
+	log.Infof("implementing the Controller")
+	mh.implementingControllerV3()
+
+	log.Infof("scaffolding webhook")
+	err = mh.ctx.CreateWebhook(
+		"--group", mh.ctx.Group,
+		"--version", mh.ctx.Version,
+		"--kind", mh.ctx.Kind,
+		"--defaulting",
+		"--defaulting")
```

Comment:
```suggestion
		"--programmatic-validation")
```

Reply:
Currently, the sample works with `defaulting `. If we change it for v3 we need to change it for v2. Otherwise, we are unable to compare both to check what changed besides increasing the changes/mock/test that cannot be re-used both at this moment. But we can do it in a follow up. Track it in:

Reply:
You've specified `--defaulting` twice